Extensions/ExternalTfs/Src/Tasks/DownloadArtifactsTfsGit/gitwrapper.js

```diff
@@ -70,7 +70,7 @@ var GitWrapper = (function (_super) {
         git.on('debug', function (message) {
             if (options.debugOutput) {
                 var repl = message.replace(creds, '...');
-                repl = message.replace(escapedCreds, '...');
+                repl = repl.replace(escapedCreds, '...');
```

Comment:
was this a bug, so we replaces message with repl?

Reply:
Yes